doc/README.md

```diff
@@ -270,6 +270,18 @@ wait for the profile.
   profile over http. If not specified, pprof will use heuristics to determine a
   reasonable timeout.
 
+pprof also accepts options which allow a user to specify TLS certificates to
```

Comment:
Could you rework the first paragraph in this section to mention this capability. Also, can we mention https+insecure thing as well? Basically I think the first paragraph needs a bit of holistic update to make it more of an overview with subsequent paragraphs diving into the details.

Reply:
I made a very small change to the first paragraph, but I'm having a hard time figuring this documentation -- I think I might not understand http protocols and pprof's use cases well enough right now to make this clear.

Reply:
"https+insecure" is not a protocol, it's a hack to disable the server's certificate validation on the client side.

Reply:
Done.